<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Confirmation</class>
 <widget class="QMainWindow" name="Confirmation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>381</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="Filename1"/>
     </item>
     <item>
      <widget class="QListWidget" name="FileContent1"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>411</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="Filename2"/>
     </item>
     <item>
      <widget class="QListWidget" name="FileContent2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>440</y>
      <width>309</width>
      <height>46</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="EqualBtn">
       <property name="text">
        <string>等价</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DoubtBtn">
       <property name="text">
        <string>存疑</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="InequalBtn">
       <property name="text">
        <string>不等价</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
